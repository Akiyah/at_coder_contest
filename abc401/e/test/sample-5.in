200 300
1 2
1 3
1 4
1 5
1 6
1 7
1 8
1 9
1 10
1 11
1 12
1 13
1 14
1 15
1 16
1 17
1 18
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 27
1 28
1 29
1 30
1 31
1 32
1 33
1 34
1 35
1 36
1 37
1 38
1 39
1 40
1 41
1 42
1 43
1 44
1 45
1 46
1 47
1 48
1 49
1 50
1 51
1 52
1 53
1 54
1 55
1 56
1 57
1 58
1 59
1 60
1 61
1 62
1 63
1 64
1 65
1 66
1 67
1 68
1 69
1 70
1 71
1 72
1 73
1 74
1 75
1 76
1 77
1 78
1 79
1 80
1 81
1 82
1 83
1 84
1 85
1 86
1 87
1 88
1 89
1 90
1 91
1 92
1 93
1 94
1 95
1 96
1 97
1 98
1 99
1 100
1 184
1 117
19 196
19 146
162 200
162 163
17 153
17 172
17 131
17 77
17 58
17 32
55 103
55 79
55 107
55 183
55 64
55 115
55 109
85 164
70 130
14 61
41 95
124 180
84 90
84 106
102 160
102 117
102 136
95 107
95 168
36 170
146 160
96 109
96 172
73 178
73 148
73 194
73 103
73 144
151 186
68 119
68 147
68 195
62 199
62 194
62 164
28 184
7 107
7 62
4 136
4 158
64 160
58 139
58 92
58 153
58 124
60 137
119 189
8 142
8 134
8 10
33 49
33 191
33 196
99 161
93 159
93 185
66 138
66 181
67 123
120 170
22 169
22 107
22 101
22 63
150 173
150 198
159 195
15 87
30 67
30 91
43 155
43 160
54 104
54 64
168 178
23 67
23 124
9 118
9 117
9 186
9 119
9 112
9 73
185 194
185 188
122 164
122 161
12 79
139 143
117 186
117 184
117 151
46 156
46 113
46 67
77 152
187 199
6 9
6 59
49 67
49 120
49 81
49 185
180 183
180 196
128 185
115 189
115 127
142 159
82 129
82 172
18 145
18 53
48 139
5 21
136 172
108 163
108 120
152 185
45 144
32 151
11 45
51 70
144 180
135 195
35 135
25 97
25 128
25 77
98 119
78 167
78 136
156 167
21 146
21 155
134 150
50 54
121 151
65 141
37 94
37 124
16 74
13 134
81 161
81 85
61 72
27 160
184 189
72 197
72 82
186 188
163 167
52 150
52 190
3 96
3 150
69 197
138 150
138 164
20 86
20 149
20 143
44 154
44 49
131 171
101 146
88 91
10 92
143 153
160 180
89 181
161 176
24 68
24 54
91 144
91 170
34 195
57 163
74 105
74 141
79 99
79 91
75 84
189 199
76 101
39 108
109 175
53 79
63 127
