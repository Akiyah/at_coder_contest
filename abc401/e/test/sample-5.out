101
100
100
101
100
99
99
100
103
102
101
100
99
98
97
96
98
98
99
100
100
101
101
100
100
99
99
98
97
96
95
95
95
95
95
95
94
93
93
92
91
90
89
89
89
90
89
89
90
89
88
88
87
87
90
89
89
88
87
87
86
88
88
87
87
88
88
88
88
88
87
86
87
87
86
85
85
85
84
83
83
83
82
82
81
80
79
78
77
76
75
74
74
73
73
72
71
70
69
68
67
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
-1
